src/UserSpaceInstrumentation/UserSpaceInstrumentationTestLib.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef USER_SPACE_INSTRUMENTATION_TEST_LIB_H_
+#define USER_SPACE_INSTRUMENTATION_TEST_LIB_H_
+
+#include <string_view>
+
```

Comment:
> I also removed the namespace in the lib; I'd like to have this as easy to work with as possible (nice short symbol names).

Maybe add a comment around here with this explanation so that the reason why the namespace is missing doesn't get lost.

Reply:
As mentioned below: This will change quite a bit no matter what and it really doesn't matter what namespace this is in since it will not link to anything.